docs/src/pages/discover-more/roadmap/roadmap.md

```diff
@@ -44,57 +38,65 @@ Here are the top priorities:
 - 0.2 - **Bundle size.** 📦 You can keep track of our progress [following bundlephobia.com report](https://bundlephobia.com/result?p=@material-ui/core).
   It's a continuous effort – v4 was [18% smaller](https://bundlephobia.com/result?p=@material-ui/core@3.9.2) than v3, while adding new features.
   We are eager to find new bundle size reduction opportunities. We hope we can leverage these two in the future:
-  - A JSS to styled-component migration should help. Compare [JSS v10.0.0-alpha.24](https://bundlephobia.com/result?p=react-jss@10.0.0-alpha.24) with [styled-components v5.0.0-beta.8](https://bundlephobia.com/result?p=styled-components@5.0.0-beta.8). We could save ~6 kB gzipped.
-  - Popper.js is working [on a smaller v2](https://bundlephobia.com/result?p=popper.js@2.0.0-next.4) version. We could save ~4 kB gzipped.
+- 0.2 - **TypeScript.** 📏 There are two dimensions to this problem:
+  - The definitions. We are **continuously** improving them. For instance, we are working on moving the props descriptions to TypeScript, so you can access them directly from your IDE.
+    The codebase is written in JavaScript, we don't plan on migrating it to TypeScript in the near future. Upvote [#15984](https://github.com/mui-org/material-ui/issues/15984) if you want us to rewrite the core in a future version.
 - 0.2 - **Accessibility.** ♿️ We have relatively [few accessibility issues](https://darekkay.com/blog/accessible-ui-frameworks/), but we are eager to address them all. We would appreciate the help of accessibility experts.
 - 0.2 - **Material Design Update.** 🎀 We are relatively up-to-date but the material design specification [is evolving](https://material.io/whats-new/), so should we.
 
 ## New components
 
-Here are the components we will work on being supported in the Material-UI ecosystem, no matter if they are from @material-ui/core, a third-party project or a paid source:
+Here are the components we will work on being supported in the Material-UI ecosystem, no matter if they are part of the [community edition](https://github.com/mui-org/material-ui) (MIT license) or [Material-UI X](https://github.com/mui-org/material-ui-x), a paid extension for enterprises:
 
-- 🧪 In the lab, in progress to move into the core
-- 🛠 Work in progress, will likely land in the lab
+- 🧪 Close to becoming stable, already released as unstable
+- 🛠 Work in progress, will be or already released as unstable
 - ⏳ Planning to build
-- ⭐️ We will consider making some of the features paid (non MIT)
+- ⭐️ We will make some of the features available in [Material-UI X](https://github.com/mui-org/material-ui-x) (non-MIT)
```

Comment:
This symbols says "new" to me.  

```suggestion
- 💵 We will make some of the features available in [Material-UI X](https://github.com/mui-org/material-ui-x) (non-MIT)
```
Or:
🔐 💳 💎 📦 ?

Reply:
✨ - sparkly and new?
⚡ - possibly, although does it suggest "paid" to you?
💎 - says "premium", and is associated with value. Could be an option, although not to be confused with Diamond Sponsors.

Reply:
I would vote for ⚡, but not strong preference. I think that we could ask the designers to come with something to differentiate the different pages in the left side navigation as well on the overall layout.